stand/libsa/libsa.3

```diff
@@ -597,6 +597,7 @@ string after the device specification is written.
 .It Xo
 .Ft int
 .Fn devinit void
+.Xc
```

Comment:
```suggestion
.Xc
.Pp
```
This is also needed for consistency with the other items.

Reply:
Thanks!  Fixed in my git, but I've removed the changes to that file out of this PR.

I also removed the changes to `nvmecontrol.8`.

This PR is now a "no changes at all to the rendered output"; the next PR will have the fixes I removed (and other output-changing fixes).